core/ItemDatabase.lua

```diff
@@ -0,0 +1,35 @@
+--[[
+AdiBags - Adirelle's bag addon.
+Copyright 2010-2022 Adirelle (adirelle@gmail.com)
+All rights reserved.
+
+This file is part of AdiBags.
+
+AdiBags is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+AdiBags is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with AdiBags.  If not, see <http://www.gnu.org/licenses/>.
+--]]
+
+local addonName, addon = ...
+
+addon.ItemDatabase = addon:NewModule('ItemDatabase', 'AceEvent-3.0')
+local db = addon.ItemDatabase
+
+function db:ReagentData(slotData)
+  if not slotData.isCraftingReagent then return false end
+  if not slotData.classID == LE_ITEM_CLASS_TRADEGOODS then return false end
```

Comment:
`LE_ITEM_CLASS_TRADEGOODS` has been deprecated.

Reply:
Resolving this since this has changed.